Documentation/project-docs/profiling-api-status.md

```diff
@@ -43,4 +43,26 @@ The notes below will help you determine what profiling APIs are safe to use. The
 ##APIs definitely known not to work yet
 ###Windows
 ###Linux
+
+* ICorProfilerInfo:
+ * `SetEnterLeaveFunctionHooks`
+
+* ICorProfilerInfo2:
+ * `SetEnterLeaveFunctionHooks2`
+
+* SetEventMask Flags that currently result in undefined behavior :
+ * `COR_PRF_DISABLE_ALL_NGEN_IMAGES`
```

Comment:
Alright, just tested it, and indeed this is now fixed. I also see you're merging it into rel/1.0.0 so I'll push another commit to this PR removing this.